<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_add_torrent</class>
 <widget class="QDialog" name="Dialog_add_torrent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>76</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add MagnetLink</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="btn_cancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btn_add">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLineEdit" name="lnEdt_mlink"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
